### PR TITLE
DM-38835: Change to project-wide configuration, part 1

### PR DESCRIPTION
**Pull request by pkubanek**
- Applied Forces limits
- Moved configuration to a single file
- Simulate mirror movement in elevation
- Fixed tests

### CODE REVIEW

File: src/LSST/M1M3/SS/States/StandbyState.cpp [hunk 1]
```diff
@@ -50,11 +53,25 @@ States::Type StandbyState::update(UpdateCommand* command) {
...
 
     Model::get().loadSettings(command->getConfigurationOverride());
-    M1M3SSPublisher::instance().getEventConfigurationApplied()->version =
-            SettingReader::instance().getSettingsVersion();
-    M1M3SSPublisher::instance().getEventConfigurationApplied()->otherInfo = "";
+
+    auto configurationApplied = M1M3SSPublisher::instance().getEventConfigurationApplied();
+    configurationApplied->configurations = "_init";
+    configurationApplied->version = GIT_HASH;
+    configurationApplied->url = CONFIG_URL;
```
Comment (mareuter) [inline src/LSST/M1M3/SS/States/StandbyState.cpp:63]: This needs to be extended to where the configuration lives within the storage location, in this case, the support system repo. It also includes all the way down to the schema directory. So, `CONFIG_URL` should be: `https://github.com/lsst-ts/ts_m1m3support/SettingFiles/v1`

**Review by mareuter — APPROVED** [review]
Couple of things to fix, but otherwise this looks good.

**Review by dmills-lsst — APPROVED** [review]
OK

File: src/LSST/M1M3/SS/States/StandbyState.cpp [hunk 2]
```diff
@@ -50,11 +53,25 @@ States::Type StandbyState::update(UpdateCommand* command) {
...
     SettingReader::instance().getSafetyControllerSettings()->ForceController.exitBumpTesting();
+    RaisingLoweringInfo::instance().sendUpdates(true);
 
     Model::get().loadSettings(command->getConfigurationOverride());
-    M1M3SSPublisher::instance().getEventConfigurationApplied()->version =
-            SettingReader::instance().getSettingsVersion();
-    M1M3SSPublisher::instance().getEventConfigurationApplied()->otherInfo = "";
+
+    auto configurationApplied = M1M3SSPublisher::instance().getEventConfigurationApplied();
+    configurationApplied->configurations = "_init";
```
[the remark this answers is not in the document]
Comment (pkubanek) [reply src/LSST/M1M3/SS/States/StandbyState.cpp:61]: Fixed. Thanks. I know that needs to be changed to _${LSST_SITE}, but I need to work on loading config on top of existing _init entries.

**Pull request merged**